server/src/modules/users/dto/users.dto.ts

```diff
@@ -87,52 +89,107 @@ export class GetUserByIdDto {
   id: string;
 }
 
-export class UpdateUsersDto extends OmitType(CreateUserDto, [
-  "password",
-  "email",
-  "isVerified",
-] as const) {}
-
 export class QueryUserDto {
   @ApiProperty()
-  @IsNumber()
   perpage: number;
 
   @ApiProperty()
-  @IsNumber()
```

Comment:
Pourquoi avoir supprimer ça?

Reply:
avec le décorateur @IsNumber() on reçoit cette erreur a l'appel de la requête 
{
    "message": [
        "limit must be a number conforming to the specified constraints"
    ],
    "error": "Bad Request",
    "statusCode": 400
}

du coup j'ai retiré

Reply:
il faut ajouter @IsOptional()